src/kernel/include/shared/cos_types.h

```diff
@@ -414,6 +432,20 @@ struct cos_stack_freelists {
  * COMP_INFO_TMEM_STK_RELINQ != 0.  Change the defines, or change the assembly" */
 /* #endif */
 
+/* Maybe this should be somewhere else... */
+struct cos_ulinvstk_entry {
+	capid_t sinv_cap;
+	vaddr_t sp;
+} __attribute__((packed));
```

Comment:
Packing this doesn't result the alignments getting wonky?

Reply:
Note that we really want to make sure that the 64 bit value is in memory that is 64 bit aligned. x86 will generate slow code for non-aligned accesses.